src/main/java/seedu/address/model/pet/Sex.java

```diff
@@ -35,7 +35,13 @@ public static boolean isValidSex(String test) {
 
     @Override
     public String toString() {
-        return value;
+        if (value.equalsIgnoreCase("f")) {
+            return "Female";
+        } else if (value.equalsIgnoreCase("m")) {
+            return "Male";
+        } else {
+            return "Unknown";
+        }
```

Comment:
Thanks for adding this to handle different formats for Sex. Perhaps instead, we should handle this at the parsing level, so that when we store, it will always be consistent? ie storage will always show as "Male" and "Female" accordingly. This will make our data more consistent

Reply:
Alright, that makes sense, I have restore the toString() method to only return sex.value and handled the conversion of sex to full form at the parsing level. Also, the parser will now check all userInput for pet and capitalise the first letter of each word. Eg. golden retriever -> Golden Retriever